src/game.js

```diff
@@ -258,6 +259,15 @@ Game.prototype.variant = function() {
 	return this._initialPosition.variant();
 };
 
+Game.prototype.tags = function(key, value) {
+	if (arguments.length === 0) {
+		return this._tags;
+	} else if (arguments.length === 1) {
```

Comment:
I debated using the tags underneath the supported tags and can do that. I want to keep the tags so that unsupported tags that show up a lot are not dropped, like I see PlyCount a lot, and some other ones and rather than adding addition interfaces beyond the Seven Tag Roster (Event, Site,...), just trust that we'll not drop any tags that are formatted correctly.

Reply:
I have renamed and reworked the code - but kept the "extra headers" so as not to lose any information that was present in the input PGN file.

Reply:
I am now only saving "extra" headers that aren't already supported in the base Game object. Hopefully that clears any confusion, and keeps any extra headers as they were passed in the input PGN.

Reply:
This is better, but in my opinion, this is still error prone. For example, if someone does this:
```
game.additionalHeaders('Event', 'some value');
```
... then it ends up with the event being defined twice in the data-structure, with possibly two different values.

The way I see that, it would be better to keep `_additionalHeaders` private and hidden within the Game class, allowing only the pgnRead/pgnWrite methods to access it. This way, it would avoid inconsistency such as a header being defined twice, it would also avoid compatibility issues if getters/setters are added for new headers in a future version of the library.

Reply:
Ok, private works for me as long as the PGN read/write don't lose data.